src/components/LinkButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link, Reset } from '@radix-ui/themes'
+import { ComponentProps } from 'react'
+
+export function LinkButton(props: ComponentProps<'button'>) {
```

Comment:
Looks like it does the same as `TextButton` https://github.com/grafana/k6-studio/blob/d14fe686a7c3bc5c8a4377969c1b876079eee2b4/src/components/TextButton.tsx#L2

Though your implementation is cleaner, can we remove the old one and use yours in both places instead?

Reply:
I will keep the `TextButton` name but replace the implementation. In retrospect, `LinkButton` sounds like a button that is a link.